iceoryx2-services/discovery/src/service_discovery/service.rs

```diff
@@ -203,6 +253,9 @@ pub struct Config {
     /// The maximum number of samples the subscriber retains in its buffer.
     pub max_buffer_size: usize,
 
+    /// The maximum number of samples the subscriber retains in its response buffer.
```

Comment:
In this context we speak of the `Client` as the receiving port/endpoint and not the subscriber. Please always use the link notation when referring to other classes, so please write ``[`Client`]``